.changeset/unlucky-peaches-join.md

```diff
@@ -0,0 +1,5 @@
+---
+"@swisspost/design-system-documentation": patch
+---
+
+ Add package type information to the components pages #5984
```

Comment:
We usually start change set messages with a past tense verb and end them with a period.
By the way, what does “#5984” mean?